Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py

```diff
@@ -647,6 +657,86 @@ def list_asset_internet_exposure_command(
                 "value": [has_active_external_services],
             }
         )
+    if asm_id_list:
```

Comment:
Wondering if we should create helper to clean up some of this code (and at least mandate for new additions).  Possible example:
`def append_search_param(search_params, field, operator, value):
    search_params.append({
        "field": field,
        "operator": operator,
        "value": value
    })

if asm_id_list:
    append_search_param(search_params, "asm_id_list", "in", str(asm_id_list).split(","))`

Reply:
Makes sense. Updated the code with this suggestion.